src/slack_insights/gui/utils/query_engine.py

```
@@ -0,0 +1,235 @@
+
+			params = json.loads(content.strip())
+
+			# Ensure all fields exist
+			return {
+				"assigner_name": params.get("assigner_name"),
+				"status": params.get("status"),
+				"recent_days": params.get("recent_days"),
+				"keywords": params.get("keywords", []),
+			}
+
+		except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)